cpp/src/parquet/column_reader.cc

```diff
@@ -386,6 +441,12 @@ std::shared_ptr<Page> SerializedPageReader::NextPage() {
       throw ParquetException("Invalid page header");
     }
 
+    EncodedStatistics page_statistics;
```

Comment:
Renaming it to data_page_statistics? For now it is out of the scope of data page, so we'd state clearly that it is meaningful for data page only.

Reply:
Done.